### ISSUE TITLE
VS Code accessibility options popup is not read aloud by Windows Narrator

### ISSUE THREAD

**Issue by dgrechka** [microsoft/vscode#136228]
**monaco-editor version:** https://vscode-web-test-playground.azurewebsites.net/
**Browser:** Edge Version 95.0.1020.30 (Official build) (64-bit)
**OS:** Microsoft Windows [Version 10.0.19043.1288]
**Steps to reproduce:**

1. Run the Windows Narrator.
2. Open the  https://vscode-web-test-playground.azurewebsites.net/
3. Focus the Monaco
4. Press Alt+F1. (This opens f VS Code accessibility options popup in Edge)

**Expected**: Narrator reads the content of VS Code accessibility options popup.
**Actual**: Narrator does not read the content of VS Code accessibility options popup.

**Comment by dgrechka** [comment]
@alexdima , FYI we have this issue for the Monaco Editor 0.92.1 (not web VS Code)

Exactly the same repro works at https://microsoft.github.io/monaco-editor/

**Comment by isidorn** [comment]
Narrator does not work well with Monaco Editor and VS Code and because of that we recommend NVDA for our windows users. More about that can be found here https://code.visualstudio.com/docs/editor/accessibility

We have this issue to track Narrator limitations https://github.com/microsoft/vscode/issues/10729
Narrator team is following that issue so it is better to have everything in one place
Thus closing this as dup of that one and have updated that one to have this case.
